src/main/java/com/zaxxer/hikari/pool/PoolElf.java

```diff
@@ -281,15 +284,6 @@ void resetConnectionState(final PoolBagEntry poolEntry) throws SQLException
       }
    }
 
-   void resetPoolEntry(final PoolBagEntry poolEntry)
-   {
-      poolEntry.setReadOnly(isReadOnly);
-      poolEntry.setCatalog(catalog);
-      poolEntry.setAutoCommit(isAutoCommit);
-      poolEntry.setNetworkTimeout(networkTimeout);
-      poolEntry.setTransactionIsolation(transactionIsolation);
-   }
-
```

Comment:
To me it is much simpler now, earlier code was HARD to follow having call to resetPoolEntry() from constructor and that method was calling methods on self to set each member.